data/aerial-photography/index.html

```diff
@@ -100,9 +101,9 @@ <h3 id="DOQ">
     </a>
     <ul class="dotless">
       <li>Black and White
-      <li>~1 meter pixels
+      <li>~1-meter pixels
     </ul>
-    <p>Digital Orthophoto Quads are typically Black and White aerial photos with a 1 meter resolution. In some cases the DOQs are .5 meter, color or 4-band. The boundaries of the imagery tiles correspond to the boundaries of 1:24,000 USGS topographic maps. The 1990's collection is statewide.</p>
+    <p>Digital orthophoto quads (DOQ) are typically black-and-white aerial photos with a 1-meter resolution. In some cases the DOQs are 0.5 meter, color, or 4-band. The boundaries of the imagery tiles correspond to the boundaries of 1:24,000 US Geological Society (USGS) topographic maps. The 1990s collection is statewide.</p>
```

Comment:
(This applies to the "DOQ Orthophotography Interactive Map" link text under the Download Links section): Should this just be "DOQ Interactive Map"?

Reply:
let's stick with 'Digital Orthophoto Quads (DOQ)'